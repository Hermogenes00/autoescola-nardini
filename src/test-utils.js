import React from "react";
import ThemeProvider from "../src/styles/ThemeProvider";
import { render } from "@testing-library/react";

const TestProvider = ({ children }) => (
  <ThemeProvider theme="light">{children}</ThemeProvider>
);

export * from "@testing-library/react";

const customRender = (ui, options) =>
  render(ui, { wrapper: TestProvider, ...options });

export { customRender as render };
