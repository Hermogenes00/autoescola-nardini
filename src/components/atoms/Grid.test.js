import React from "react";
import { render } from "test-utils";

import Grid from "./Grid";

test("match snapshot  with no params", () => {
  const { asFragment } = render(<Grid></Grid>);

  expect(asFragment()).toMatchSnapshot();
});
