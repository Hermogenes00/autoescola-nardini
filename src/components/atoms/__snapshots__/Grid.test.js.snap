// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot  with no params 1`] = `
<DocumentFragment>
  .c0 {
  display: grid;
  grid-column-gap: 16px;
  grid-row-gap: 16px;
  grid-template-columns: 1fr;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
